intro-to-rails/rails-forms.md

```diff
@@ -182,19 +184,126 @@ The above ERB code generates this HTML:
 
 In `BooksController#new`, give the book instance variable a default title.  Do you notice a change in the form?
 
-  <details>
-    <summary>Solution</summary>  
-    The form's title field should now display the text which was set in the controller.
-  </details>
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
```

Comment:
This question on line 185 is the same as the first Exercise (question on line 200) right? Might want to delete the one on 185

Reply:
👍